### ISSUE TITLE
Functionality not correct for 1 model & 2 diagnostics

### ISSUE THREAD

**Issue by jonnyhtw** [pletzer/rosesnip#1]
Hi there

I have just tried to run `rosesnip` for one model (u-az513) and two diagnostics, `tas_global` and `tas_tropics`.

The files in `tmp` are generated correctly...

```
> pwd
/home/williamsjh/rosesnip/results/tmp/nc/awmean
> ls -l
total 512
-rw-rw-r-- 1 williamsjh williamsjh 17150 May  5 00:09 az513_apy_m01s00i024_global_0.nc
-rw-rw-r-- 1 williamsjh williamsjh 17245 May  5 00:09 az513_apy_m01s00i024_tropics_1.nc
```

... but the final data is being combined into one file rather than two...

```
> pwd
/home/williamsjh/rosesnip/results/nc/awmean
> ls -l
total 256
-rw-rw-r-- 1 williamsjh williamsjh 39793 May  5 00:09 az513_apy_m01s00i024_global.nc
```

This appears to be due to an incorrect call to `cdo mergetime`...

```
> ncdump -h az513_apy_m01s00i024_global.nc |grep -i history
		:history = "Tue May 05 00:09:46 2020: cdo mergetime 
/scale_wlg_persistent/filesets/home/williamsjh/rosesnip/results/tmp/nc/awmean/az513_apy_m01s00i024_global_0.nc 
/scale_wlg_persistent/filesets/home/williamsjh/rosesnip/results/tmp/nc/awmean/az513_apy_m01s00i024_tropics_1.nc 
/scale_wlg_persistent/filesets/home/williamsjh/rosesnip/results/nc/awmean/az513_apy_m01s00i024_global.nc\n2020-05-05T00:09:41Z: Afterburner v1.3.1b3: Statistical data generated by the Climate Model Monitor v2 application." ;
```

Please let me know what I can do to assist here in testing.

Many thanks

Jonny

**Comment by jonnyhtw** [comment]
Update

I've now tried it with `tas_global` and `sw_rad_flux` and it works as expected for one model.

Therefore the issue appears to lie with the fact that the two original files in the issue had a common prefix, that is...

`az513_apy_m01s00i024_`

I guess that the `cdo mergetime` call needs to take this into account?

I'll take a look at the code. 😄 

Cheers

Jonny

**Comment by jonnyhtw** [comment]
Update 

I've found the bit of the code. I think it needs two further loops.

1. Over `_land_`, `_sea_` and neither.
2. Over region name, e.g. `_global_` and `tropics`.

Cheers

![image](https://user-images.githubusercontent.com/12940715/81029788-682cc280-8eda-11ea-83a4-dd29b856540f.png)

**Comment by pletzer** [comment]
Thanks @jonnyhtw. It looks like we cannot assume that the merge operation takes place over time....Looking into this.

**Comment by jonnyhtw** [comment]
> Thanks @jonnyhtw. It looks like we cannot assume that the merge operation takes place over time....Looking into this.

OK thanks @pletzer. Sounds good. 😄

**Comment by pletzer** [comment]
Hi @jonnyhtw,

This has been resolved in PR #3.
